snprc_ehr/test/src/org/labkey/test/tests/snprc_ehr/SNPRC_EHRTest.java

```diff
@@ -267,6 +267,14 @@ protected void importStudy()
         importFolderFromPath(++_pipelineJobCount);
     }
 
+    @Override
+    protected List<String> skipLinksForValidation()
+    {
+        List<String> links = new ArrayList<>(super.skipLinksForValidation());
+        links.add("snprc_ehr-BirthReport.view");
```

Comment:
Leave a comment explaining why this is skipped.

Reply:
done